fn/common/funcfile.go

```diff
@@ -43,6 +43,7 @@ type Funcfile struct {
 	Memory         *int64            `yaml:"memory,omitempty" json:"memory,omitempty"`
 	Format         *string           `yaml:"format,omitempty" json:"format,omitempty"`
 	Timeout        *time.Duration    `yaml:"timeout,omitempty" json:"timeout,omitempty"`
+	IDLETimeout    *time.Duration    `yaml:"idle_timeout,omitempty" json:"idle_timeout,omitempty"`
```

Comment:
nits: I think IdleTimeout is better than IDLETimeout.

Reply:
Yes but unfortunately it is defined as `IDLETimeout` in [functions_go](https://github.com/iron-io/functions_go/blob/master/models/route.go#L33) (which is auto generated). I am not sure why it capitalizes.

Reply:
Oh, so it's no problem.
Thanks 👍